sandbox/mock/helloworld.json

```diff
@@ -1,3 +1,3 @@
 {
-  "message": "Hello World!"
-}
\ No newline at end of file
+  "message": "Test2!"
```

Comment:
I believe the spec defines what these should be (hello world, hello application, hello user) -- make sure it coforms with the spec-generated documentation